docs/root/development/performance/battery_impact.rst

```diff
@@ -0,0 +1,63 @@
+.. _dev_performance_battery:
+
+Analysis of battery impact
+==========================
+
+In order to identify how Envoy impacts an application, we have created a control application with modifications to our
+current hello world example applications. To see the actual applications we used, you can go to `iOS Envoy example <https://github.com/lyft/envoy-mobile/tree/ac/envoy-battery-cpu-branch/examples/swift/hello_world>`_,
+`Android control app <https://github.com/lyft/envoy-mobile/tree/ac/envoy-battery-cpu-branch/examples/kotlin/control>`_, `Android Envoy example <https://github.com/lyft/envoy-mobile/tree/ac/envoy-battery-cpu-branch/examples/kotlin/hello_world>`_.
+
+The current configurations we have are that we make a request every 200ms and disabled caching.
```

Comment:
```suggestion
The current configurations make network requests every 200ms and disable caching.
```

Reply:
We should also probably fill in more details here regarding how Envoy is set up

Reply:
Do you mean that Envoy is a process currently and we are making requests to Envoy via a socket...etc?

I think we can move that more towards the specific platform descriptions because iOS and Android has specific nuances that impact how we are using Envoy today. 

What do you think?

Reply:
Right, I primarily wanted to point out that these tests were performed while using native networking + Envoy rather than calling into Envoy directly. For example, these results will likely be different on iOS when we stop using `URLSession`. Can add in a separate PR though